src/App.js

```diff
@@ -1,23 +1,186 @@
-import logo from './logo.svg';
-import './App.css';
+import { useEffect, useState } from 'react';
+import './App.scss';
+import ship from './assets/ship.png';
+
+import image100 from './assets/image100.png';
+import image200 from './assets/image200.png';
+import image300 from './assets/image300.png';
+import image400 from './assets/image400.png';
+import image500 from './assets/image500.png';
+
 
 function App() {
+  // Creating states
+  const [question, setQuestion] = useState("");
+  const [rightAnswer, setRightAnswer] = useState("")
+  const [selectedValue, setSelectedValue] = useState(null);
+  const [finalValue, setFinalValue] = useState("");
+  const [score, setScore] = useState(0);
+  const [formSubmitted, setFormSubmitted] = useState(false);
+  const [gameStarted, setGameStarted] = useState(0);
+
+  // Fetching the api Data
+  useEffect(() => {
+
+    async function getData() {
+      try {
+        const response = await fetch("https://opentdb.com/api.php?amount=1&type=boolean");
+        const data = await response.json();
+        setRightAnswer(data.results[0].correct_answer);
+        setQuestion(data.results[0].question);
+      } catch (error) {
+        console.error(error);
+      }
+    }
+
+    getData();
+
+  }, [gameStarted])
+
+
+
+  // console.log(question);
```

Comment:
Very small thing but you may want to remove the commented out console.logs to save some lines of code again (I got dinged for leaving commented out code in my project 2)